NAM/util.cpp

```diff
@@ -8,4 +8,11 @@ std::string util::lowercase(const std::string& s)
   std::string out(s);
   std::transform(s.begin(), s.end(), out.begin(), [](unsigned char c) { return std::tolower(c); });
   return out;
-}
\ No newline at end of file
+}
+
+void util::init_matrix(Eigen::MatrixXf& matrix)
```

Comment:
I believe there's an Eigen function for this...can we use [`setZero`](https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#title95) here?

In fact, would it make sense to scrap this function altogether and replace those changes with `.setZero()`s?

Reply:
Good catch! Somehow I couldn't find it in the docs. All fixed now.